Sources/LiveKit/Core/Room.swift

```diff
@@ -28,11 +28,19 @@ public class Room: NSObject, ObservableObject, Loggable {
 
     // MARK: - Public
 
-    @objc
+    @objc(sid)
     /// Server assigned id of the Room.
     /// This will be empty until ``RoomDelegate/room(_:didUpdateRoomId:)`` is called.
     public var sid: Sid? { _state.sid }
 
+    // Work-around error: Property with 'throws' or 'async' is not representable in Objective-C
+    /// Server assigned id of the Room. async version of ``Room/sidValue``.
+    public var asyncSid: Sid {
```

Comment:
how would one use this value? `await room.asyncSid()` ? should we state usage in the comment?

Reply:
Xcode will show it's an await-able property.